main.js

```diff
@@ -18,6 +18,13 @@ window.addEventListener('load', () => {
 });
 document.getElementById('addTasks').addEventListener('click', function() {
     addTodo(todos);
+    // get the title input
+    const todoTitleInput = document.getElementById('todo-title');
+    // get the due date input
+    const dueDateInput = document.getElementById('date-picker');
+    // clear the inputs
+    todoTitleInput.value = "";
+    dueDateInput.value = "";
```

Comment:
Try doing this on your own without copying. Challenge yourself to see if you can get right.

Don't forget to create a feature branch.